inc/cleantalk-settings.php

```diff
@@ -2061,6 +2061,27 @@ function apbct_settings__validate($settings)
         return $settings;
     }
 
+    // Ajax type
+    $available_ajax_type = apbct_settings__get_ajax_type();
+
+    if (!isset($apbct->data['ajax_type'])) {
+        $apbct->data['ajax_type'] = $available_ajax_type;
+    }
+
+    if (
+        (isset($settings['data__set_cookies']) && $settings['data__set_cookies'] == 2) ||
+        (isset($settings['data__use_ajax']) && $settings['data__use_ajax'] == 1)
+    ) {
```

Comment:
**true && false || true** is not the same as **true && ( false || true )**

Please, use the parenthesis in the right place.

Reply:
My bad. It is ok.

But, anyway, no "isset" is needed